website/pages/docs/getting-started/vue.mdx

```diff
@@ -0,0 +1,172 @@
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm create vite@latest test-app -- --template vue-ts
+    cd test-app
+    npm install
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn create vite test-app --template vue-ts
+    cd test-app
+    yarn
+    ```
+  </Tab>
+</Tabs>
+
+
+### Install Panda
+
+Install panda and create your `panda.config.ts` file.
+
+<Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tab>
+    ```bash
+    pnpm install -D @pandacss/dev
+    pnpm panda init --postcss
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm install -D @pandacss/dev
+    npx panda init --postcss
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn add -D @pandacss/dev
+    yarn panda init --postcss
+    ```
+  </Tab>
+</Tabs>
+
+### Update package.json scripts
+
+Open your `package.json` file and update the `scripts` section as follows:
+
+```diff {3} filename="package.json"
+{
+  "scripts": {
++    "prepare": "panda codegen",
+    "dev": "vite",
+    "build": "vue-tsc && vite build",
+    "lint": "eslint src --ext ts,tsx,vue --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  }
+}
+```
+- `"prepare"` - script that will run Panda CSS CLI codegen before each build. Read more about [codegen](/docs/references/cli#panda-codegen) in the CLI section.
+
+### Configure the content
+
+Make sure that all of the paths of your Vue components are included in the `include` section of the `panda.config.ts` file.
+
+```js {8, 17} filename="panda.config.ts"
+import { defineConfig } from "@pandacss/dev"
+
+export default defineConfig({
+ // Whether to use css reset
+ preflight: true,
+
+ // Where to look for your css declarations
+ include: ["./src/**/*.{js,jsx,ts,tsx,vue}", "./pages/**/*.{js,jsx,ts,tsx,vue}"],
+
+ // Files to exclude
+ exclude: [],
+
+ // The output directory for your css system
+ outdir: "styled-system",
+})
+```
+
+### Configure the entry CSS with layers
+
+Add this code to an `src/style.css` file imported in the root component of your project.
+
+```css filename="src/style.css"
+@layer reset, base, tokens, recipes, utilities;
+```
+
+> **Note:** Feel free to remove the contents of `<style>` tag inside `src/App.vue` file as we don't need it anymore.
+
+## Start your build process
+
+Run the following command to start your development server.
+
+<Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tab>
+    ```bash
+    pnpm dev
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    npm run dev
+    ```
+  </Tab>
+  <Tab>
+    ```bash
+    yarn dev
+    ```
+  </Tab>
+</Tabs>
+
+### Start using Panda
+
+Now you can start using Panda CSS in your project.
+Here is the snippet of code that you can use in your `src/App.vue` file.
+
+<Tabs items={['SFC', 'JSX']}>
+  <Tab>
+    ```vue filename="src/App.vue"
+    <script setup lang="ts">
+    import { css } from '../styled-system/css';
+    </script>
+
+    <template>
+      <div :class="css({ fontSize: '2xl', fontWeight: 'bold' })">Hello 🐼!</div>
+    </template>
+    ```
+  </Tab>
```

Comment:
Panda can only work in Vue JSX syntax since it parses only JS(X)/TS(X) files for now.

Let's remove this section and add a note about SFC in Vue, which is not supported for now.

Reply:
It works on SFC! Unless Im missing something?

I’ll try to build the app and check if I get an error. Either way, will update the PR later.

Reply:
Panda works 100% with SFC @segunadebayo. Do you know a specific Panda feature that is not supported?

<img width="727" alt="Screenshot 2023-06-18 at 9 26 27 PM" src="https://github.com/chakra-ui/panda/assets/13049130/65c866a1-d70d-4ec9-aa04-08723326b399">

Reply:
@wobsoriano could you check the browser and see if the styles were applied. They won't be.

Reply:
I've tried and it doesn't seem to work in my browser in build or dev. As @wobsoriano i saw the cli updates

Reply:
Yes, this is happening because our static analysis engine doesn't support Vue language syntax at the moment. We're going to support that in the future.

For now, I'd recommend putting the styles in a dedicated `<component>.css.ts` file and declaring the styles there, pretty much like CSS Modules.

This breaks co-location but we'll definitely improve the DX

Reply:
So, yeah. The styles weren't applied. How did I miss that? 😆 

And for the usage, I added a note about JSX being supported for now.